ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -60,48 +60,55 @@ func Ps(c *cli.Context) {
 	if err := options.ValidateFlagPairs(c); err != nil {
 		logrus.Fatal(err.Error())
 	}
-	containers, err := listContainers(c)
+	containers, pathname, err := listContainers(c)
 	if err != nil {
-		logrus.Fatalf("Failed to list containers due to:\n%v", err)
+		logrus.Fatalf("Failed to list containers for %s due to:\n%v", pathname, err)
 	}
-	if err = displayContainers(c, containers); err != nil {
-		logrus.Fatalf("Failed to display containers due to:\n%v", err)
+	if err = displayContainers(c, pathname, containers); err != nil {
+		logrus.Fatalf("Failed to display containers for %s due to:\n%v", pathname, err)
 	}
 }
 
-func listContainers(c *cli.Context) ([]types.Container, error) {
+func listContainers(c *cli.Context) ([]types.Container, string, error) {
```

Comment:
The return types of this function is not obvious anymore. From this function I'd expect only `([]types.Container, error)`. It's not easy to tell what he `string` in the middle represents.

Reply:
Instead of returning the input string can we instead just log it in this function?  Something like:
```go
logrus.Infof("Searching for containers matching --%s=%s", flags.TaskDefinitionFile, c.String(flags.TaskDefinitionFile))
```